hedera-node/test-clients/src/main/java/com/hedera/services/bdd/spec/utilops/FakeNmt.java

```diff
@@ -38,7 +38,92 @@ public static TryToStartNodesOp restartNetwork(
         requireNonNull(envOverrides);
         return new TryToStartNodesOp(
-                NodeSelector.allNodes(), configVersion, SubProcessNode.ReassignPorts.YES, envOverrides);
+                NodeSelector.allNodes(), configVersion, SubProcessNode.ReassignPorts.NO, envOverrides);
+    }
+
+    /**
+     * Returns an operation that restarts the network with the given config version without
+     * refreshing override-network.json files.
+     *
+     * @param configVersion the config version to use
+     * @param envOverrides the environment overrides to use
+     * @return the operation that restarts the network
+     */
+    public static TryToStartNodesOp restartNetworkNoOverride(
+            final int configVersion, @NonNull final Map<String, String> envOverrides) {
+        requireNonNull(envOverrides);
+        return new TryToStartNodesOp(
+                NodeSelector.allNodes(), configVersion, SubProcessNode.ReassignPorts.NO, envOverrides, false);
+    }
+
+    /**
+     * Returns an operation that restarts the network with the given config version without
+     * refreshing override-network.json files.
+     *
+     * @param configVersion the config version to use
+     * @return the operation that restarts the network
+     */
+    public static TryToStartNodesOp restartNetworkNoOverride(final int configVersion) {
+        return restartNetworkNoOverride(configVersion, Map.of());
+    }
+
+    /**
+     * Returns an operation that starts the selected nodes with the given config version.
+     *
+     * @param selector the selector for the nodes to start
+     * @param configVersion the config version to use
+     * @param envOverrides the environment overrides to use
+     * @return the operation that starts the nodes
+     */
+    public static TryToStartNodesOp startNodes(
+            @NonNull final NodeSelector selector,
+            final int configVersion,
+            @NonNull final Map<String, String> envOverrides) {
+        requireNonNull(selector);
+        requireNonNull(envOverrides);
+        return new TryToStartNodesOp(selector, configVersion, SubProcessNode.ReassignPorts.NO, envOverrides);
+    }
+
+    /**
+     * Returns an operation that starts the selected nodes with the given config version without
+     * refreshing override-network.json files.
+     *
+     * @param selector the selector for the nodes to start
+     * @param configVersion the config version to use
+     * @param envOverrides the environment overrides to use
+     * @return the operation that starts the nodes
+     */
+    public static TryToStartNodesOp startNodesNoOverride(
+            @NonNull final NodeSelector selector,
+            final int configVersion,
+            @NonNull final Map<String, String> envOverrides) {
+        requireNonNull(selector);
+        requireNonNull(envOverrides);
+        return new TryToStartNodesOp(selector, configVersion, SubProcessNode.ReassignPorts.NO, envOverrides, false);
+    }
```

Comment:
If we've really removed the dependency on override-networks.json, it's surprising that  'do X' and 'do X No Override' variants are referenced so frequently throughout the PR 🤔 I expected overrides to be more encapsulated